### PR TITLE
inline data

### PR DESCRIPTION
**Pull request by adeel-usmani**

### CODE REVIEW

**Comment by adeel-usmani** [comment]
Brother @UmanShahzad can you please review this?

**Review by UmanShahzad — COMMENTED** [review]
We're gonna wanna do a backwards-incompat change and just accept an in-memory map from the user. See some of the other SDKs for example. Forget about accepting a 'file' now.

File: src/IPinfo.php [hunk 1]
```diff
@@ -16,6 +16,1017 @@
  */
 class IPinfo
 {
+    const CONTINENTS_DEFAULT = [
```
Comment (UmanShahzad) [inline src/IPinfo.php:19]: Pls let's put it in some other file, this ends up filling the main logic file with a huge amount of data that must be scrolled through

**Review by UmanShahzad — APPROVED** [review]

**Comment by jfiel** [comment]
Hi Team,

You can do whatever you want but you should know that moving the constants to a file that is required instead of putting them in a class or trait caused our application which uses PHP preload to error during during runtime.

The error at runtime was "Undefined constant "ipinfo\ipinfo\COUNTRIES".

I had to write an exclusion to prevent ipinfo\ipinfo\IPinfo from being compiled with my application in preload.

**Pull request merged**